sycl/source/context.cpp

```diff
@@ -23,25 +23,31 @@
 
 __SYCL_INLINE namespace cl {
 namespace sycl {
-context::context(const async_handler &AsyncHandler)
-    : context(default_selector().select_device(), AsyncHandler) {}
+context::context(const async_handler &AsyncHandler, bool usePrimaryContext)
```

Comment:
Nit:
```suggestion
context::context(const async_handler &AsyncHandler, bool UsePrimaryContext)
```

Same applies to the rest of the file.